src/Aquifer.Jobs/readme.md

```diff
@@ -1,6 +1,6 @@
 # Aquifer.Jobs
 
-We have two kinds of Azure Function jobs, Subscriber and Managers.  Both kinds are triggered and run asynchronously.
+We have two kinds of Azure Function jobs, MessageSubscribers and Managers.  Both kinds are triggered and run asynchronously.
```

Comment:
I fixed a readme typo as part of this PR in order to have something to deploy. The post-merge action won't run for only workflow changes (though I could change that).